docs/dev_guide/code_standards.rst

```diff
@@ -51,11 +81,30 @@ The categories include:
 * [E]rror for important programming issues (i.e. most probably bug)
 * [F]atal for errors which prevented further processing
 
-PyLint checks a wide range of different things so the first time you run PyLint
-on a file you will likely get a large number of warnings. In some cases the
-warnings will help you to spot coding mistakes or areas that could be improved
-with refactoring. In other cases, however, the warning message may not apply
-and what you have there is exactly as it should be. In these cases it is
-possible to silence PyLint for that line. PyLint warning messages can be
-disabled at three different levels: globally (using a .pylintrc file),
-file-wide, and for a single line.
+PyLint checks a wide range of different things so the first time you run PyLint on a file you will likely get a large number of warnings. In some cases the warnings will help you to spot coding mistakes or areas that could be improved with refactoring.
+In other cases, however, the warning message may not apply and what you have there is exactly as it should be.
+In these cases it is possible to silence PyLint for that line.
+PyLint warning messages can be disabled at three different levels: globally (using a .pylintrc file), file-wide, and for a single line.
+
+Porting Code from IDL
```

Comment:
I feel like this section definately needs a re-write

Reply:
I'ma gonna delete it. I don't think we want to encourage IDL to Python porting directly